iot-hub/Samples/service/DeviceStreamingSample/Parameters.cs

```diff
@@ -0,0 +1,36 @@
+﻿using CommandLine;
+using Microsoft.Azure.Devices;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ServiceClientStreamingSample
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'c',
+            "HubConnectionString",
+            Required = true,
+            HelpText = "The connection string of the IoT Hub instance to connect to.")]
+        public string HubConnectionString { get; set; }
+
+        [Option(
+            'd',
+            "DeviceId",
+            Required = true,
+            HelpText = "The Id of the device to connect to.")]
+        public string DeviceId { get; set; }
+
+        [Option(
+            't',
+            "TransportType",
+            Default = TransportType.Amqp,
```

Comment:
Can we default to MQTT?

Reply:
The TransportType enum in our service client does not have MQTT.  Are the different protocols only for the device client?

Reply:
Oops, good point.